src/networks.json

```diff
@@ -108,6 +108,21 @@
     "explorer": "https://ubiqscan.io",
     "logo": "ipfs://Qmec3HLoN4QhwZAhw4XTi2aN8Wwmcko5hHN22sHARzb9tw"
   },
+  "4181": {
+    "key": "4181",
+    "name": "PHI",
+    "chainId": 4181,
+    "network": "PHI",
+    "multicall": "0xB141139f17c9F6B6B0da82bfC8E1153570AD44fE",
+    "rpc": [
+      "https://rpc1.phi.network"
```

Comment:
Hey @Phinetwork Looks like this is not a full archive node?? try passing older block numbers

Reply:
It give this error:
```
missing trie node b2409a4e98f07e15137e31d89255a9707cbd45ab07baefb2c540c46702851208
```

Reply:
Node Provided is a full archive node.

Reply:
Can you try calling it with older block numbers like 10, 100, 1000 etc